### PR TITLE
Add [0..a) and [1..a]

### PR DESCRIPTION
**Pull request by nayakrujul**

### CODE REVIEW

File: shared/src/vyxal/Elements.scala [hunk 1]
```diff
@@ -1515,6 +1513,38 @@ object Elements:
       case a: VNum => a ** 3
       case a: String => VList.from(a.grouped(3).toSeq)
     },
+    addPart(
+      Monad,
+      "ɾ",
+      "Inclusive One Range | Uppercase",
+      List("1..n", "one->n", "one-range", "to-upper", "upper", "uppercase"),
```
Comment (Lyxal) [inline shared/src/vyxal/Elements.scala:1520]: I didn't think `.` was valid in literate mode keywords.

**Review by Lyxal — APPROVED** [review]
Cool beans homie. Cool beans.

File: shared/src/vyxal/Elements.scala (hunk 1 above)
Comment (nayakrujul) [reply shared/src/vyxal/Elements.scala:1520]: Umm... well it didn't give me any errors :p

Comment (Lyxal) [reply shared/src/vyxal/Elements.scala:1520]: Well of course not

There's not tests or checks for that

Comment (nayakrujul) [reply shared/src/vyxal/Elements.scala:1520]: Fine. I'll change it

**Pull request merged**